exporters/metric/stdout/stdout.go

```diff
@@ -132,14 +131,16 @@ func InstallNewPipeline(config Config, opts ...push.Option) (*push.Controller, e
 
 // NewExportPipeline sets up a complete export pipeline with the recommended setup,
 // chaining a NewRawExporter into the recommended selectors and integrators.
-func NewExportPipeline(config Config, period time.Duration, opts ...push.Option) (*push.Controller, error) {
-	selector := simple.NewWithExactDistribution()
+func NewExportPipeline(config Config, options ...push.Option) (*push.Controller, error) {
 	exporter, err := NewRawExporter(config)
 	if err != nil {
 		return nil, err
 	}
-	integrator := integrator.New(selector, true)
-	pusher := push.New(integrator, exporter, period, opts...)
+	pusher := push.New(
+		simple.NewWithExactDistribution(),
+		exporter,
+		append(options, push.WithStateful(true))...,
```

Comment:
Changing to forcing stateful for the `stdout` exporter seems unwarranted.

Reply:
I made it the default, but allowed it to be overridden. This was the old behavior.